pyfar/classes/coordinates.py

```diff
@@ -2891,13 +2911,13 @@ def __rdiv__(self, other):
 
 
     def _check_empty(self):
-        """check if object is empty."""
+        """Check if object is empty."""
         if self.cshape == (0,):
             raise ValueError('Object is empty.')
 
 
 def dot(a, b):
-    """Dot product of two Coordinates objects.
+    r"""Dot product of two Coordinates objects.
 
     .. math::
         \\vec{a} \\cdot \\vec{b}
```

Comment:
The formula isn't displayed correctly in the docs, probably because of the double backslashes.